bin/end.rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
demo private key file, also only temporary. Will use the enclave private key in the future (#364 )